gst/nnstreamer/tensor_repo/tensor_repo.h

```diff
@@ -141,7 +104,7 @@ gst_tensor_repo_set_changed (guint o_nth, guint nth, gboolean is_sink);
  * @brief Get GstTensorRepoData from repo.
  */
 GstBuffer *
-gst_tensor_repo_get_buffer (guint nth, gboolean *eos, guint *newid);
+gst_tensor_repo_get_buffer (guint nth, gboolean *eos, guint *newid, GstCaps ** caps);
```

Comment:
Minor comment: `GstCaps **caps` to keep the same format?

Reply:
better to change other params :)